app/models/survey_response.rb

```diff
@@ -46,4 +48,22 @@ def upsert_responses!(new_responses)
     upsert_responses(new_responses)
     save!
   end
+
+  # Helper method to generate CSV
+  def self.to_csv(survey_responses)
+    CSV.generate(headers: true) do |csv|
+      csv << ["ID", "User ID", "Stop", "Responses", "Created", "Updated"]
```

Comment:
I think `updated_at` can be removed from this CSV file. It should always be identical to `created_at`. Also—tricky detail—the response column's keys need to be extracted and added to the header. However, this is further complicated by the fact that a response from six months ago might have different questions than a response today. Thus, you'll have to extract all of the responses' keys, `uniq` them, and then match up keys and values. Let me know if you have questions about this!

Reply:
Done